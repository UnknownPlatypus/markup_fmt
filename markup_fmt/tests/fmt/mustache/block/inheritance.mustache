{{$ title    }}={{/    title }}

{{$foo}}default {{#bar}}{{baz}}{{/bar}} content{{/foo}}

{{<include  }}{{/include }}

{{<parent}} {{$block}}
      one
      two{{/block}}
      {{/parent}}
