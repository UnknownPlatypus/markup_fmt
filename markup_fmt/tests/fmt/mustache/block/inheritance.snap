---
source: markup_fmt/tests/fmt.rs
---
{{$title}}={{/title}}

{{$foo}}default {{#bar}}{{baz}}{{/bar}} content{{/foo}}

{{<include}}{{/include}}

{{<parent}}
  {{$block}}
    one two{{/block}}
{{/parent}}
